proto/regen/ecocredit/v1/tx.proto

```diff
@@ -35,11 +35,20 @@ service Msg {
   rpc MintBatchCredits(MsgMintBatchCredits)
       returns (MsgMintBatchCreditsResponse);
 
-  // MsgSealBatch sets the `batch.open` attribute to false. Sealed batch
+  // SealBatch sets the `batch.open` attribute to false. Sealed batch
   // can't issue more credits. Once batch is sealed it can't be toggled any
   // more. Only batch creator can seal a batch.
   rpc SealBatch(MsgSealBatch) returns (MsgSealBatchResponse);
 
+  // MintPendingBatchCredits creates a pending batch issuance of credits for an issuer to approve
+  // within a specified amount of time.
+  rpc MintPendingBatchCredits(MsgMintPendingBatchCredits) returns (MsgMintPendingBatchCreditsResponse);
+
+  // ApproveBatchCredits approves a pending batch of credits, delivering the credit issuances
+  // to the users specified in the initial MintPendingBatchCredits RPC call. Only the designated
+  // issuer for a given set of pending batch credits can approve them.
+  rpc ApproveBatchCredits(MsgApproveBatchCredits) returns (MsgApproveBatchCreditsResponse);
```

Comment:
`ApprovePendingBatch`?

Reply:
I like robert's suggestion on the name changes